### PR TITLE
Fixes for link issues

### LINKED ISSUE #1431: Broken links
1. all guides under -> https://docs.metamask.io/wallet are throwing Page not found
> It seems to be a trailing slash issue:
> https://docs.metamask.io/wallet doesn't work
> https://docs.metamask.io/wallet/ does
2. On Pull > yarn install > yarn start of main, the following links aren't resolved:
-  (../reference/polygon/index.md) in source file "/Users/{user}/Documents/repos/metamask-docs/services/concepts/websockets.md" for version current
- (../../dashboard/create-api.md) in source file "/Users/{user}//Documents/repos/metamask-docs/services/tutorials/ethereum/send-erc-20-token-rust.md" for version current
- (../reference/ethereum/json-rpc-methods/eth_call.mdx) in source file "/Users/{user}//Documents/repos/metamask-docs/services/reference/_partials/_eth_estimategas-description.mdx" for version current
- (../../concepts/transaction-types.md) in source file "/Users/{user}//Documents/repos/metamask-docs/services/reference/_partials/trace-methods/_trace_call-parameters.mdx" for version current

### PR DESCRIPTION
**Pull request by m4sterbunny**
# Description

Begins fixes for broken links. NB it appears there is 1. a trivial issue of path not existing and 2. of valid path in markdown not rendering as valid path in build >> first commit fixes type 1

## Issue(s) fixed

Fixes #1431 

## Preview

none

## Checklist

Complete this checklist before merging your PR:

- [ ] If this PR contains a major change to the documentation content, I have added an entry to the top of the ["What's new?"](https://github.com/MetaMask/metamask-docs/blob/main/docs/whats-new.md) page.
- [ ] The proposed changes have been reviewed and approved by a member of the documentation team.

### CODE REVIEW

**Comment by joaniefromtheblock** [comment]
Thanks Harrie! For the homepage, I think the issue is something else. Sometimes the links are resolving to the correct spot and other times the link is missing the baseURL "wallet". So, when I click on "Connect" sometimes it goes to "https://docs.metamask.io/wallet/how-to/connect" and then other times it goes to "https://docs.metamask.io/how-to/connect/". Looking into this further

**Comment by m4sterbunny** [comment]
Hmm, interesting -- the trailing slash fix works locally for me, but not with the vercel build

File: docusaurus.config.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ const config = {
     },
     { src: "/js/feedback-script.js", defer: true, async: true },
     { src: "/js/getfeedback.js", defer: true, async: true },
+    { src: '/js/trailingSlash.js', defer: true, async: true},
```
[the remark this answers is not in the document]
Comment (joaniefromtheblock) [reply docusaurus.config.js:50]: Not incorrect, just looking at consistency with the others. I think we are using double quotes

File: wallet/index.mdx [hunk 2]
```diff
@@ -12,24 +12,24 @@ You can interact with your users' Ethereum accounts, performing tasks such as th
...
     },
     {
-      href: "how-to/display",
+      href: "how-to/display/",
       title: "📄 Display in MetaMask",
       description: "Display tokens, contract methods, and icons in MetaMask.",
     },
     {
-      href: "how-to/use-sdk",
+      href: "how-to/use-sdk/",
```
[the remark this answers is not in the document]
Comment (joaniefromtheblock) [reply wallet/index.mdx:32]: I think your script is enough. I don't think slashes needed to be added here

**Comment by m4sterbunny** [comment]
closed in favor of https://github.com/MetaMask/metamask-docs/pull/1436

**Pull request closed without merging**